src/chains/ethereum/ethereum/src/wallet.ts

```diff
@@ -119,7 +120,16 @@ export default class Wallet {
   readonly unlockedAccounts = new Map<string, Data>();
   readonly lockTimers = new Map<string, NodeJS.Timeout>();
 
-  constructor(opts: EthereumInternalOptions["wallet"]) {
+  constructor(
+    opts: EthereumInternalOptions["wallet"],
+    logging: EthereumInternalOptions["logging"]
+  ) {
+    // checkForConflicts in @ganache/options will prevent
```

Comment:
I don't understand this comment. Can you elaborate or remove?

Reply:
:+1: Yup, partial note to myself. 

I did not want to create a condition where the message could fire unnecessarily. I was originally doing the check in `#initializeAccounts` but `checkForConflicts` will prevent a user from specifying both given accounts and a `totalAccounts`. So it should be safe in the constructor and reduce passing the logger around more than is needful.